roles/docker/templates/docker-options.conf.j2

```diff
@@ -1,2 +1,7 @@
 [Service]
-Environment="DOCKER_OPTS={% if docker_options is defined %}{{ docker_options }}{% endif %}"
+{% if is_atomic_host  %}
+Environment="OPTIONS=\
+{% else %}
+Environment="DOCKER_OPTS=\
+{% endif %}
+{% if docker_options is defined %}{{ docker_options }}{% endif %}"
```

Comment:
This looks dirty and not readable. I'd rather repeat this line on lines 3 and 5 than rely on an escaped line break